src/rules.mk

```diff
@@ -120,15 +121,27 @@ $(VALE):
 	mkdir -p $(TOOLS_DIR)
 	curl -L $(VALE_URL)/$(VALE_LINUX) -o $(TOOLS_DIR)/$(VALE_LINUX)
 	cd $(TOOLS_DIR) && tar -xzf $(VALE_LINUX)
+	$(MAKE) install-vale-styles-linux
 endif
 
 ifeq ($(UNAME),Darwin)
 $(VALE):
 	mkdir -p $(TOOLS_DIR)
 	curl -L $(VALE_URL)/$(VALE_MACOS) -o $(TOOLS_DIR)/$(VALE_MACOS)
 	cd $(TOOLS_DIR) && tar -xzf $(VALE_MACOS)
+	$(MAKE) install-vale-styles-macos
 endif
 
+install-vale-styles-linux:
+	mkdir -p $(STYLE_DIR)
+	curl --location https://github.com/errata-ai/proselint/archive/v0.3.2.tar.gz | tar -C $(STYLE_DIR) -xz --strip-components=1 --wildcards '*/proselint'
+	curl --location https://github.com/errata-ai/write-good/archive/v0.4.0.tar.gz | tar -C $(STYLE_DIR) -xz --strip-components=1 --wildcards '*/write-good'
```

Comment:
please re-wrap these to fit them into a 79 char line length limit

Reply:
also: the URLs used her should probably be defined in the variables block at the start of the file (just as a matter of style, but also because they are repeated elsewhere)